generation/regenerate-core-api.sh

```diff
@@ -2,6 +2,13 @@
 
 set -e
 
+# !!!WARNING!!!
+# OAS file must be manually changed so that:
```

Comment:
You might want to try out a python script like we have in the node - and perform these changes automatically? 👍

Reply:
I'll do it once we move SDKs into separate repo. Then I'll do my best to make sure C#, Java, TypeScript models and SDKs are generated using more or less same tooling in a consistent manner.